src/components/LibraryDetails.tsx

```diff
@@ -30,31 +26,51 @@ const LibraryDetails = ({ library }: LibraryDetailsProps) => {
     areas,
     stages,
   } = library;
-  const [libraryStage, setLibraryStage] = useState<LibraryStage>('');
-  const [registryStage, setRegistryStage] = useState<LibraryStage>('');
+  const [libraryStage, setLibraryStage] = useState<LibraryStage>(
+    stages.library_stage
+  );
+  const [registryStage, setRegistryStage] = useState<LibraryStage>(
+    stages.registry_stage
+  );
 
-  useEffect(() => {
-    setLibraryStage(stages.library_stage);
-  }, [stages.library_stage]);
+  // useEffect(() => {
```

Comment:
Stray comments? Not sure if this is intentional or if we need it

Reply:
Was thinking I might use for the next PR, but will delete for now.